test/fixtures/packages/missing-main/index.js

```diff
@@ -0,0 +1,22 @@
+// Copyright Joyent, Inc. and other Node contributors.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the
+// "Software"), to deal in the Software without restriction, including
+// without limitation the rights to use, copy, modify, merge, publish,
+// distribute, sublicense, and/or sell copies of the Software, and to permit
+// persons to whom the Software is furnished to do so, subject to the
+// following conditions:
+//
+// The above copyright notice and this permission notice shall be included
+// in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+// MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN
+// NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+// OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
+// USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+exports.ok = 'ok';
```

Comment:
:) I left it since it is identical to several other files (so that the tests are consistent in design) and at least arguably is a copy of them.

Reply:
> :) I left it since it is identical to several other files (so that the tests are consistent in design) and at least arguably is a copy of them.

Sounds good to me. Thanks!

Reply:
New files should *not* contain the notice.

Reply:
New files that, like this one, are exact copies of existing files should contain the notice.

This file is an exact copy of `test/fixtures/packages/main/package-main-module.js` and `test/fixtures/packages/main-index/package-main-module/index.js`.